README.md

```diff
@@ -55,7 +55,7 @@ int main()
     std::cout << r.equity[0] << " " << r.equity[1] << std::endl;
 }
 ```
-###Advanced usage
+### Advanced usage
 ```c++
 #include <omp/EquityCalculator.h>
```

Comment:
Shouldn't it be also `#include "omp/EquityCalculator.h"` instead of `#include <omp/EquityCalculator.h>`?

Reply:
Yes thanks!